c7n/resources/glue.py

```diff
@@ -272,3 +274,135 @@ def process(self, resources):
                 client.delete_table(DatabaseName=r['DatabaseName'], Name=r['Name'])
             except client.exceptions.EntityNotFoundException:
                 continue
+
+@resources.register('glue-classifier')
+class GlueClassifier(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'glue'
+        enum_spec = ('get_classifiers', 'Classifiers', None)
+        id = name = 'Name'
+        date = 'CreationTime'
+        arn_type = 'classifier'
+
+    permissions = ('glue:GetClassifiers',)
+
+@GlueClassifier.action_registry.register('delete')
+class DeleteClassifier(BaseAction):
+
+    schema = type_schema('delete')
+    permissions = ('glue:DeleteClassifier',)
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('glue')
+        for r in resources:
+            classifier = next(iter(r.values()))
+            try:
+                client.delete_classifier(Name=classifier['Name'])
+            except client.exceptions.EntityNotFoundException:
+                continue
+
+@resources.register('glue-ml-transform')
+class GlueMLTransform(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'glue'
+        enum_spec = ('get_ml_transforms', 'Transforms', None)
+        name = 'Name'
+        id = 'TransformId'
+        arn_type = 'mltransform'
```

Comment:
Is this value correct? From what I read in the docs, arn listed for this resource type is : `arn:${Partition}:glue:${Region}:${Account}:mlTransform/${TransformId}`
https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awsglue.html#awsglue-mlTransform